Manito/Manito/Screens/Detail-Wait/DetailWaitViewController.swift

```diff
@@ -49,183 +49,157 @@ final class DetailWaitViewController: BaseViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        self.fetchRoomData()
         self.configureDelegation()
         self.configureNavigationController()
+        self.detailWaitViewModel.fetchRoomInformation()
+        self.setBind()
```

Comment:
p2;

set + 명사 가 좋다고 생각해요! 
```suggestion
        self.setupBindings()
```
는 어떠신가용?

Reply:
좋습니다! 지난번에 호야랑 공유하면서 호야는 setBind로 하셨단 말을 얼핏 들은거 같아서 호다닥 수정했었는데, 너무 얼핏들어버렸네요..ㅋㅋ 수정했슴당 ^_________^